config/sync/block.block.recipe_theme_page_title.yml

```diff
@@ -17,4 +19,9 @@ settings:
   label: 'Page title'
   provider: core
   label_display: '0'
-visibility: {  }
+visibility:
+  request_path:
+    id: request_path
+    pages: '/user/*'
```

Comment:
If the Page Title is disabled on the user page, will that page have a `h1` element? From a SEO and Accessibility perspective it's important that a webpage has at least and preferably one `h1` element.